tests/Behat/FeatureContext.php

```diff
@@ -278,6 +278,20 @@ public function selectOption(string $select, string $option, string $region): vo
     $regionObj->selectFieldOption($select, $option);
   }
 
+  /**
+   * Selects option in select field with specified selector.
+   *
+   * @param string $element
+   *   The element selector.
+   * @param string $option
+   *   The option to select.
+   *
+   * @When I select :option from :element form element
```

Comment:
We already have this step in drupal extension.

Reply:
That uses locator, not element.